.eslintrc.js

```diff
@@ -1,21 +1,39 @@
 module.exports = {
   root: true,
   extends: ['@metamask/eslint-config', '@metamask/eslint-config-nodejs'],
-  env: {
-    commonjs: true,
-  },
+
   overrides: [
     {
-      files: ['test/**/*.js'],
+      files: ['**/*.test.ts'],
       extends: ['@metamask/eslint-config-jest'],
+      rules: {
+        'node/no-unpublished-require': 0,
```

Comment:
Curious why this is needed? Typically we only run into this issue for private packages, or packages we haven't published yet.

Reply:
I think you're right, I think this is not needed anymore.  It was needed in some interim states of this PR.